src/git-utils.js

```diff
@@ -19,8 +19,8 @@ import { GitUrl } from '@adobe/helix-shared-git';
 
 import git from 'isomorphic-git';
 // cache for isomorphic-git API
-// see https://isomorphic-git.org/docs/en/cache
-const cache = {};
+// Note: cache disabled for compatibility with isomorphic-git 1.33.x
```

Comment:
@stefan-guggisberg there seems to be some incompatibility and disabling the cache looks like a brutal workaround, maybe too brutal